app/src/main/java/org/mozilla/fenix/components/Core.kt

```diff
@@ -319,6 +324,14 @@ class Core(
 
     val pinnedSiteStorage by lazyMonitored { PinnedSiteStorage(context) }
 
+    @Suppress("MagicNumber")
+    val pocketConfig by lazyMonitored {
+        PocketStoriesConfig(
+            BuildConfig.POCKET_TOKEN, client, Frequency(4, TimeUnit.HOURS), 7
+        )
+    }
+    val pocketUpdater by lazyMonitored { PocketStoriesService(context, pocketConfig) }
```

Comment:
nit: I have been trying to find the "pocketUpdater" implementation in code, until I read this line and noticed it's actually a `PocketStoriesService`. Maybe those should be named similar?